src/invoices/InvoiceDetails/InvoiceDetails.css

```diff
@@ -8,3 +8,7 @@
   height: 23px;
   text-align: center;
 }
+
+.updateOrderStatusModalBody {
+  margin-top: 12px;
```

Comment:
As opposed to writing ui-module level CSS, please use the[ `<Layout>` component ](https://github.com/folio-org/stripes-components/tree/master/lib/Layout)
It's our basic way of providing style utilities for common things like offsets/padding etc. You can use the page in our storybook to find the applicable className to use for what you need to adjust: https://dev.folio.org/stripes-components/?path=/story/components-layout--available-classes